### PR TITLE
[Feature 3] created view profile screen

### LINKED ISSUE #3: View Profiles
view new profiles to like or dislike them

### PR DESCRIPTION
**Pull request by candidcalcifer**
1/2 of view profile issue

### CODE REVIEW

File: app/src/main/java/com/courseproject/tindar/presenters/viewprofiles/ViewProfilesPresenter.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.courseproject.tindar.presenters.viewprofiles;
+
+public class ViewProfilesPresenter {
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/presenters/viewprofiles/ViewProfilesPresenter.java:3]: If you don't need this class, you can delete this file and the folder it belongs to. If you are planning to add something here later, pls add a TODO comment.

**Comment by jenniferjinyoungyang** [comment]
Please add Javadocs. If you want to focus on other implementations first, please create a tech debt ticket to finish Javadocs for this feature.

**Review by jenniferjinyoungyang — CHANGES_REQUESTED** [review]

**Review by navarhontes — CHANGES_REQUESTED** [review]
Don't fully understand TextView and MutableLiveData (especially its use in the case of Birthday since Birthday shouldn't change), but I'll defer to you since I don't know how much they care about the quality of frontend code? Other than that, same comments as Jennifer. JavaDocs also makes it easier to give good PR reviews.

**Review by jenniferjinyoungyang — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Review by navarhontes — APPROVED** [review]
Approving because it seems to work and that's the current priority. Made issues out of the remaining problems to be addressed later.

**Pull request merged**